src/test/java/nextstep/subway/acceptance/LineSteps.java

```diff
@@ -64,4 +64,16 @@
                 .when().delete("/lines/{lineId}/sections?stationId={stationId}", lineId, stationId)
                 .then().log().all().extract();
     }
+
+    public static Long 지하철_노선_생성_요청(String name, String color, Long upStation, Long downStation, int distance) {
+        Map<String, String> lineCreateParams;
+        lineCreateParams = new HashMap<>();
```

Comment:
라인을 굳이 나눠서 초기화 하신 이유가 있을까요? 🤔 
```suggestion
        Map<String, String> lineCreateParams = new HashMap<>();
```